FreydCategoriesForCAP/gap/CategoryOfGradedRowsAndColumns/CategoryOfGradedColumns.gi

```diff
@@ -435,7 +435,7 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_CAP_CATEGORY_OF_GRADED_COLUMNS,
     # @Returns a morphism
     # @Arguments zero_object, object
```

Comment:
Probably this line should be changed, too.

Reply:
The lines are not used to generate a documentation via autodoc (yet). So this description does not influence the functionality.

However, you are right. For reasons of consistency this should be changed. I have just done so.

Thanks for pointing this out.